### PR TITLE
chore(browserslist): Use browserslist-config-monday on Vibe

### PR DESCRIPTION
**Pull request by YossiSaadi**
https://monday.monday.com/boards/3532714909/pulses/5097839339

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -249,16 +250,7 @@
...
-    "Firefox ESR",
-    "not IE 11",
-    "Edge 18",
-    "not dead",
-    "not Samsung > 0",
-    "not op_mini all",
-    "not and_ff > 0",
-    "not and_qq > 0",
-    "not and_uc > 0"
+    "extends browserslist-config-monday"
```
Comment (orrgottlieb) [inline package.json:253]: this is a breaking change....

Comment (shlomitc) [reply package.json:253]: If I understand correctly - this is the exact same configuration we are using in the Monday production, so it's basically just an alignment

Comment (YossiSaadi) [reply package.json:253]: @orrgottlieb I agree with @shlomitc
It means this is a breaking change only for consumers that aren't using vibe to build monday apps

**Review by shlomitc — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (orrgottlieb) [reply package.json:253]: this is still an open source library, which we have other users... @shlomitc @YossiSaadi 

open source is open source, you should go by the community standards for semver

**Pull request merged**